snippets/serverless-about-kafka-broker.adoc

```diff
@@ -0,0 +1,22 @@
+// Text snippet included in the following modules and assemblies:
+//
+// * /modules/serverless-broker-types.adoc
+// * /serverless/develop/serverless-kafka-developer.adoc
+
+:_content-type: SNIPPET
+
+For production-ready Knative Eventing deployments, Red Hat recommends using the Knative Kafka broker implementation. The Kafka broker is an Apache Kafka native implementation of the Knative broker, which sends CloudEvents directly to the Kafka instance.
+
+[IMPORTANT]
+====
+The Kafka broker is not supported on FIPS.
+====
+
+The Kafka broker has a native integration with Kafka for storing and routing events. This allows better integration with Kafka for the broker and trigger model over other broker types, and reduces network hops. Other benefits of the Kafka broker implementation include:
+
+* At-least once delivery guarantees
```

Comment:
Are all these words as intended? The sentence seems off to me. My first thought was that "once" was a typo, but maybe not.

Reply:
Yeah they are :) but it should probably be "at-least-once". It refers to a specific delivery method: see https://doc.akka.io/docs/akka/current/general/message-delivery-reliability.html?language=scala